### PR TITLE
SN-API-3 Add migrations

### PR DESCRIPTION
**Pull request by Aretemal**

### CODE REVIEW

File: config/config.js [hunk 1]
```diff
@@ -1,2 +0,0 @@
-const config = { secret: 'SECRET_KEY_RANDOM' };
-export default config;
```
Comment (Konstantsin) [inline config/config.js:2]: Why did u remove this?

File: index.js [hunk 2]
```diff
@@ -13,15 +13,14 @@ const PORT = process.env.SECRET_PORT || 5000;
 
 app.use(cors()); // Use this after the variable declaration
 app.use(express.json());
-app.use(errorHandler);
 app.use('/api', router);
-
+app.use(errorHandler);
```
Comment (Konstantsin) [inline index.js:17]: It looks like it's a part of other pull request.

File: config/config.js (hunk 1 above)
Comment (Aretemal) [reply config/config.js:2]: This file is in folder Config

File: index.js (hunk 2 above)
Comment (Aretemal) [reply index.js:17]: I moved this line to the end of all middleware.

**Review by Konstantsin — APPROVED** [review]

**Pull request merged**